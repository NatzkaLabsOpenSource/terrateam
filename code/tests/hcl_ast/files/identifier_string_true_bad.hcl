foo "bar" 1
---
foo "bar" true
---
foo "bar" false
---
foo "bar" 1.2
