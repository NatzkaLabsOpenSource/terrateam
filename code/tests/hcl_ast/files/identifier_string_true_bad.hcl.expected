Expected block labels, which are strings or identifiers, found a non-string literal.
